.github/workflows/ci_web.yml

```
@@ -2,6 +2,16 @@ name: ci-web
 on:
   workflow_call:
 jobs:
+  validate-nginx-config:
+    # TODO: update the referer to tag or commit hash
+    uses: reearth/actions/.github/workflows/validate-nginx-config.yml@feat/nginx-config-validator/1
+    env:
```

actionlint (1.7.4)

when a reusable workflow is called with "uses", "env" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "validate-nginx-config"

(syntax-check)